packages/runner-ct/cypress/component/SpecList.spec.tsx

```diff
@@ -147,19 +148,26 @@ describe('SpecList', () => {
 
     // find via folder + file combination. rp from merp, cat.ts from cat.spec.ts.
     cy.realType('rpcat.ts')
-
     cy.get('div').contains('foo.spec.js').should('not.exist')
     cy.get('div').contains('dog.spec.tsx').should('not.exist')
     cy.get('div').contains('cat.spec.ts').should('exist')
 
-    // the found characters, cat.ts, should be bold via <b>
-    ;['r', 'p', 'c', 'a', 't', '.', 't', 's'].forEach((char) => {
-      cy.get('[data-item="merp/cat.spec.ts"]').get('b').contains(char)
+    // Wait to ensure typing has completed and React has rerendered
+    cy.wait(10).then(() => {
+      // the found folder characters, rp
+      ['r', 'p'].forEach((char) => {
+        cy.get('[title="merp"] > div > span span').contains(char)
+      })
+
+      // the found file characters, ct.ts, should be bold via <span>
+      ;['c', 'a', 't', '.', 't', 's'].forEach((char) => {
+        cy.get('[title="merp/cat.spec.ts"] > span span').contains(char)
```

Comment:
I wonder if we actually need `> span span`. Seems a bit fragile - does this add any certainty to the results of the test? Not really a big deal, still going to ✅ , just curious.

Reply:
Without grabbing the class assigned to certain nodes, yes, it is necessary. I believe the previous test actually wasn't specific enough, as it would find other highlighted characters outside of this row.